sigopt_sklearn/search.py

```diff
@@ -227,9 +243,9 @@ def _convert_log_params(self, param_dict):
       log_converted_dict = {}
       for pname in param_dict:
         pval = param_dict[pname]
-        if "__log__" in pname:
+        if '__log__' in pname:
           pval = math.exp(pval)
-          pname = pname.replace("__log__", "")
+          pname = pname.replace('__log__', '')
         log_converted_dict[pname] = pval
       return log_converted_dict
```

Comment:
Do we document anywhere that this log transformation happens automatically?  I couldn't find it, but I didn't look that hard.